python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -116,8 +124,13 @@ def start(self, options: ExecutionOptions):
         super().start(options)
 
         # Create the actor workers and add them to the pool.
-        self._cls = ray.remote(**self._ray_remote_args)(_MapWorker)
         for _ in range(self._autoscaling_policy.min_workers):
+            remote_args = self._ray_remote_args
+            if self._scheduling_strategy_fn:
+                # For each new actor, get new scheduling strategy by
+                # calling the generation fn.
+                remote_args["scheduling_strategy"] = self._scheduling_strategy_fn()
+            self._cls = ray.remote(**remote_args)(_MapWorker)
```

Comment:
We should also handle the actor created in scale up, right? https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/execution/operators/actor_pool_map_operator.py#L241-L247